lib/tasks/development.rake

```diff
@@ -53,6 +53,7 @@ unless Rails.env.production?
 
   # Run the tests and fail if coverage is below 100%
   task test_strict: :environment do
+    Dotenv.overload(".env.test")
```

Comment:
Rake tasks use .env.test by default when they are called so it needs to be overloaded. I've encountered this issue in other projects and there is a task in asana [Rake task test_all_strict uses .env.development vars instead of .env.test](https://app.asana.com/0/1142794766483633/1200878922116037/f)

Is this quick-fix good enough?